### PR TITLE
Test: Added a TC for line coverage

### PR DESCRIPTION
**Pull request by nik-redhat**
Fixes: #679, Fixes: #680

Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
Test run:
```
Table:
 test_volume_features_for_line_coverage
 
+-------------+-------------+-------------------+
| Volume Type | Test Result |  Time taken (sec) |
+-------------+-------------+-------------------+
|     dist    |     PASS    | 36.08504128456116 |
+-------------+-------------+-------------------+


Framework runtime : 45.971070528030396
```

File: common/ops/gluster_ops/bitrot_ops.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                      excep: bool = True) -> dict:
+        """
+        Enables bitrot for given volume
+
+        Args:
+            volname (str): volume name
+            node (str): Node on which cmd has to be executed.
+        Optional:
+            excep (bool): exception flag to bypass the exception if the
+                          setup volume command fails. If set to False
```
Comment (aujjwal-redhat) [inline common/ops/gluster_ops/bitrot_ops.py:25]: enable bitrot command fails : )

File: common/ops/gluster_ops/bitrot_ops.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+                       excep: bool = True) -> dict:
+        """
+        Disable bitrot for given volume
+
+        Args:
+            volname (str): volume name
+            node (str): Node on which cmd has to be executed.
+        Optional:
+            excep (bool): exception flag to bypass the exception if the
+                          setup volume command fails. If set to False
```
Comment (aujjwal-redhat) [inline common/ops/gluster_ops/bitrot_ops.py:53]: disable bitrot command fails : )

File: tests/functional/glusterd/test_volume_features_for_line_coverage.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+        cmd = ("ls -l /var/run/gluster/ | grep 'glusterdump' | wc -l")
+        ret = redant.execute_abstract_op_node(cmd, self.server_list[0])
+        if ret['msg'][0].rstrip('\n') != '1':
+            raise Exception("Failed to generate statedump for glusterd"
+                            " process")
+
+        # Replace brick with itself should fail
+        brick_list = redant.get_all_bricks(self.vol_name, self.server_list[0])
+        ret = redant.replace_brick(self.server_list[0], self.vol_name,
```
Comment (aujjwal-redhat) [inline tests/functional/glusterd/test_volume_features_for_line_coverage.py:123]: Can we have a none check as well for get_all_bricks ?

**Review by aujjwal-redhat — DISMISSED** [review]

File: tests/functional/glusterd/test_volume_features_for_line_coverage.py (hunk 3 above)
Comment (nik-redhat) [reply tests/functional/glusterd/test_volume_features_for_line_coverage.py:123]: Ack.

**Review by srijan-sivakumar — DISMISSED** [review]
LGTM

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**